api/Src/Repositories/TeamRepository.ts

```diff
@@ -39,6 +39,22 @@ export default class TeamRepository extends Repository {
     return await this.updateById(teamId, updateQuery)
   }
 
+  async addUsersToTeam(
+    teamId: mongoose.Types.ObjectId,
+    usersIds: Array<mongoose.Types.ObjectId>,
+  ) {
+    for (const userId of usersIds) {
+      const updateQuery = {
+        $push: {
+          users: userId,
+        },
+      }
```

Comment:
https://docs.mongodb.com/manual/reference/operator/update/push/
Should be possible to add all user together once, instead of looping

Reply:
sure but we still need a loop to add it to the collection because our current service doesnt provide updateMany

Reply:
done